xunit.runner.visualstudio.desktop/VsTestRunner.cs

```diff
@@ -507,7 +507,7 @@ void RunTests(IRunContext runContext, IFrameworkHandle frameworkHandle, LoggerHe
             return @event;
         }
 
-        static IRunnerReporter GetRunnerReporter(IEnumerable<string> assemblyFileNames)
+        public static IRunnerReporter GetRunnerReporter(IEnumerable<string> assemblyFileNames)
```

Comment:
Let me know if there's a different way you'd prefer to expose this for unit tests.